packages/desktop-client/src/components/budget/index.js

```diff
@@ -516,6 +516,16 @@ const RolloverBudgetSummary = memo(props => {
   );
 });
 
+const ReportBudgetSummary = memo(props => {
+  const isGoalTemplatesEnabled = useFeatureFlag('goalTemplatesEnabled');
```

Comment:
Since `BudgetSummary` is a function component, couldn’t this hook call be moved into there to avoid the wrapping component?

Reply:
The reason I did it this was because I was copying the method used in the above function since it's solving the same problem for the Rollover budget.  I can try to implement what you're asking though.